alembic/versions/d190096b7d13_async_migrations.py

```diff
@@ -0,0 +1,49 @@
+"""Async migrations
+
+Revision ID: d190096b7d13
+Revises: 
+Create Date: 2024-10-31 13:10:09.461980
+
+"""
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision: str = 'd190096b7d13'
+down_revision: Union[str, None] = None
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('city',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('name', sa.String(length=128), nullable=False),
+    sa.Column('additional_info', sa.Text(), nullable=True),
+    sa.PrimaryKeyConstraint('id'),
+    sa.UniqueConstraint('name')
+    )
+    op.create_index(op.f('ix_city_id'), 'city', ['id'], unique=False)
+    op.create_table('temperature',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('city_id', sa.Integer(), nullable=True),
+    sa.Column('date_time', sa.DateTime(), nullable=True),
+    sa.Column('temperature', sa.String(length=8), nullable=False),
```

Comment:
The `temperature` column is defined as a `String` with a length of 8. Ensure that this is sufficient for your temperature data format. If temperatures are stored as numbers, consider using a numeric type like `Float` or `Integer`.